pkg/templates/kubernetes/resources/pod.yaml

```diff
@@ -35,6 +35,22 @@ properties:
       spec:
         type: model(kubernetes:PodSpec)
 
+
+consumption:
+  consumed:
+    - model: EnvironmentVariables
+      property_path: Object.spec.containers[0].env
```

Comment:
This kinda feels like it should be just a property as the `model`, with the `converter` being the default value of `property_path`. Fine for now, but feels like it could be unified.